src/cli/telemetry/helpers.ts

```diff
@@ -12,15 +12,19 @@ export const tryFn = async <T extends (...args: any[]) => Promise<R>, R>(fn: T,
 
 export declare const TERMINAL_INFO: d.TerminalInfo;
 
-export const isInteractive = (sys: d.CompilerSystem, config: d.Config, object?: d.TerminalInfo): boolean => {
+export const isInteractive = (
+  sys: d.CompilerSystem,
+  config: d.InternalStrictConfig,
+  object?: d.TerminalInfo
+): boolean => {
   const terminalInfo =
     object ||
     Object.freeze({
       tty: sys.isTTY() ? true : false,
       ci:
         ['CI', 'BUILD_ID', 'BUILD_NUMBER', 'BITBUCKET_COMMIT', 'CODEBUILD_BUILD_ARN'].filter(
           (v) => !!sys.getEnvironmentVar(v)
-        ).length > 0 || !!config.flags?.ci,
+        ).length > 0 || !!config.flags.ci,
```

Comment:
I had to search out `config.flags?` specifically (à la grep), as `tsc` couldn't find those (maybe eslint could, I didn't deep dive that)